raiden/ui/cli.py

```diff
@@ -921,6 +917,18 @@ def smoketest(ctx, debug, local_matrix, **kwargs):  # pylint: disable=unused-arg
         'HumanStandardToken',
     )
 
+    # Check for synapse server start script if matrix is requested as transport
+    if ctx.parent.params['transport'] == 'matrix':
+        local_matrix_path = Path(local_matrix)
```

Comment:
@ulope I decided to not do these checks to support the full command-line. We may not receive a script, but a full command line for synapse, with options, and I think we could/should support that.

Reply:
@andrevmatos For that to work we would need to enable executing via shell in the `HTTPExecutor` (they all default to `shell=False`). I'm not sure if that's something anyone would really use.